distribution/docs/src/main/resources/_contents/_registry-contents/managing-registry-contents.adoc

```diff
@@ -1,26 +1,532 @@
+|true
+|yes
+
+|Source Activation Priority Order
+|sourceActivationPriorityOrder
+|String
+|This is the priority list used to determine which source should be activated on creation
+|CSW_2.0.2,WFS_1.0.0,OpenSearch_1.0.0
+|yes
+
+|===
+
+.[[Csw_Registry_Store]]Registry Store
+[cols="1,1m,1,2,1,1" options="header"]
+|===
+|Name
+|Property
+|Type
+|Description
+|Default Value
+|Required
+
+|Registry ID
+|id
+|String
+|The unique name of the store
+|
+|yes
+
+|CSW URL
+|cswUrl
+|String
+|URL to the endpoint implementing CSW spec capable of returning ebrim formatted records
+|
+|yes
+
+|Username
+|username
+|String
+|Username for CSW Service (optional)
+|
+|no
+
+|Password
+|password
+|Password for CSW Service (optional)
+|Password
+|
+|no
+
+|Allow Push
+|pushAllowed
+|Boolean
+|Enable push (write) to this registry store
+|true
+|yes
+
+|Allow Pull
+|pullAllowed
+|Boolean
+|Enable pull (read) from this registry store
+|true
+|yes
+
+|===
+
 === Using ${ddf-registry}
 
-==== Subscribing
+The *Local Registry Nodes* and *Remote Registries* tabs appear in both the ${ddf-registry} application and the ${ddf-catalog} application.
 
-===== Editing a Subscription
+==== Configuring Identity Node
 
-===== Deleting a Subscription
+* Navigate to ${ddf-registry}* (or ${ddf-catalog}) application.
+* Navigate to *Local Registry Nodes* tab.
+* Click the name of the identity node.
+* Complete all _required_ and any desired _optional_ fields.
+** Add any desired <<adding_a_service_binding,service bindings>> under the <<service_bindings, Services>> tab.
+* Click *Save*.
+** If service bindings were added, confirm setup by locating the identity node in the *Sources* tab of the *${ddf-catalog}* app.
+
+.General Information Tab
+[cols="1,3,1,1" options="header"]
+|===
+|Field
+|Description
+|Type
+|Required
+
+|Node Name
+|This node's name as it should appear to external systems
+|string
+|yes
+
+|Node Description
+|Short description for this node
+|string
+|yes
+
+|Node Version
+|This node's Version
+|string
+|yes
+
+|Security Attributes
+|Security attributes associated with this node.
+|String
+|
+
+|Last Updated
+|Date this entry's data was last updated
+|
+|
+
+|Live Date
+|Date indicating when this node went live or operational
+|
+|
+
+|Custom Fields
+a|click *Add* button (icon:plus[]) to add custom fields
+|Configurable
+|no
+
+|Associations
+a|click *Add* button (icon:plus[]) to add custom fields
+|Configurable
+|no
+
+|===
+
+.[[service_bindings]]Services
+[cols="1,3,1,1" options="header"]
+|===
+|Field
+|Description
+|Type
+|Required
+
+|Service Name
+|This service name
+|string
+|
+
+|Service Description
+|Short description for this service
+|string
+|
+
+|Service Version
+|This service version
+|string
+|
+
+|Service Type
+|Identifies the type of service this is by a urn
```

Comment:
`urn` doesn't seem right ... should it be an acronym? Like `URN`? Is it a spelling error?